docs/installation.md

```
@@ -139,3 +139,42 @@ $ uv pip install "aigverse[adapters]"
+  (`sudo apt install berkeley-abc`); it is also in conda-forge and Homebrew.
+- **Bundled toolchains** — [oss-cad-suite](https://github.com/YosysHQ/oss-cad-suite-build)
+  includes an `abc` binary.
+
+`aigverse` locates the executable in this order:
+
+1. an explicit path set via {py:func}`~aigverse.abc.set_abc_binary`,
+2. the `AIGVERSE_ABC` environment variable,
+3. `abc` or `berkeley-abc` on `PATH`.
+
+```console
+$ export AIGVERSE_ABC=/path/to/abc
```

markdownlint-cli2 (0.23.0)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)